packages/account/src/providers/operations.graphql

```diff
@@ -15,17 +15,26 @@ fragment transactionStatusFragment on TransactionStatus {
       id
     }
     time
+    receipts {
+      ...receiptFragment
+    }
     programState {
       returnType
       data
     }
+    receipts {
+      ...receiptFragment
+    }
   }
   ... on FailureStatus {
     block {
       id
     }
     time
     reason
+    receipts {
+      ...receiptFragment
```

Comment:
Because `transactionStatusFragment` is used in subscriptions as well, adding `receipts` directly onto it would mean that we'd be fetching them via `submitAndAwait`/`statusChange` subscriptions for every `Provider.sendTransaction` call and `TransactionResponse` creation even though we wouldn't be using them because we're taking the receipts from the response of `getTransactionWithReceipts`.

I suggest that you keep this `transactionStatusFragment` as you modified it, but create a separate `transactionStatusSubscriptionFragment` that'd only cover our current subscription needs. It'd look like this:

```gql
fragment transactionStatusSubscriptionFragment on TransactionStatus {
  type: __typename
  ... on SubmittedStatus {}
  ... on SuccessStatus {}
  ... on FailureStatus {}
  ... on SqueezedOutStatus {
    reason
  }
}

subscription submitAndAwait($encodedTransaction: HexString!) {
  submitAndAwait(tx: $encodedTransaction) {
    ...transactionStatusSubscriptionFragment
  }
}

subscription statusChange($transactionId: TransactionId!) {
  statusChange(id: $transactionId) {
    ...transactionStatusSubscriptionFragment
  }
}
```

I didn't try it, but maybe you could even get away with this fragment:

```gql
fragment transactionStatusSubscriptionFragment on TransactionStatus {
  type: __typename
  ... on SqueezedOutStatus {
    reason
  }
}
```


#1644 will use the receipts from `TransactionStatus` and reevaluate how receipts are integrated into `TransactionResponse`, etc.

Reply:
@nedsalk I will take a look into this

Reply:
@nedsalk I've added the receipts to the statuses fragments because starting on `0.23` receipts were removed from the root of the Transaction type on the graphQL schema and are now only present within the statuses.

Reply:
Yes, I understood that and that's why I said 
>[...] I suggest that you keep this transactionStatusFragment as you modified it...

My comment relates to keeping the fragments used in the subscriptions the same or trimming them down to only the fields used with the subscriptions in our code base currently.